src/scanner.c

```diff
@@ -359,7 +359,7 @@ static Char GetCleanedChar( UInt *wasEscaped ) {
 }
 
 
-void GetNumber ( UInt StartingStatus )
+static void GetNumber(const UInt StartingStatus)
 {
   Int                 i=0;
```

Comment:
This is a minor comment, but do we construct non-pointer arguments anywhere else? (Never noticed it). This is a possibly significant style change, so I wanted it to be intentional.

Reply:
What do you mean with "construct"? Perhaps a typo and you meant "use const argument"? The only reason to add this is to help humans reason about the code: it is now clear that the starting status variable doesn't get changed later on.

I see no significant style change with that, though?!

Reply:
Sorry, autocorrect. I meant constraint argument. It's just we could probably make a significant chunk of the arguments to C functions construction, but I thought this was the first.

Reply:
Autocorrect agsin? :-)

Anyway, I will remove the const here, and move it to the commmit it belongs to, then we can discuss more when it comes up again.